### ISSUE TITLE
Rename WG K8s Infra to SIG K8s Infra

### ISSUE THREAD

**Issue by spiffxp** [kubernetes/community#6036]
Umbrella / placeholder issue to track all of the things that need to be renamed in order to complete the conversion of WG K8s Infra to SIG K8s Infra

- [ ] proposal PR that updates sigs.yaml, updates charter
- [ ] rename slack channel
- [ ] rename mailing lists
- [ ] add github topic
- [ ] rename meeting link
- [ ] rename meeting agenda doc
- [ ] rename wg/k8s-infra github label
- [ ] rename github teams
- [ ] rename google groups used for RBAC

There's probably some things missing from that list but it's a good start

/wg k8s-infra
/priority important-soon
/kind cleanup
/milestone v1.23

/assign

**Comment by spiffxp** [comment]
/committee steering
FYI, not a thing I expect steering needs to handle, but just so they're aware

**Comment by spiffxp** [comment]
Renaming the kubernetes-wg-k8s-infra@googlegroups.com mailing list:
- [x] send notice: https://groups.google.com/g/kubernetes-dev/c/c0P08_9IXAU/m/qO-afjMvCQAJ
- [ ] update calendar invite for bi-weekly meeting
- [ ] update meeting doc permissions

**Comment by spiffxp** [comment]
Renaming the wg-k8s-infra-leads@kubernetes.io group:
- [x] Ensure the shared calendar is shared with the new list
- [ ] Ensure the zoom account uses the new e-mail
- [ ] Ensure doc permissions are updated appropriately

**Comment by spiffxp** [comment]
Job configs, testgrid dashboards and tests are all kind of intermingled.  It seemed easier to do them in a single PR: https://github.com/kubernetes/test-infra/pull/23694

**Comment by spiffxp** [comment]
I've opened https://github.com/kubernetes/k8s.io/issues/2780 as a followup issue to rename the account used for the groups reconciler. It's not public-facing so I'm less concerned with waiting on that before calling this done

**Comment by spiffxp** [comment]
/close
OK, I believe everything that can or should be renamed has been renamed.  Thanks for the help @ameukam @tehcyx @mrbobbytables
